### PR TITLE
Upgrade package.json and remove canvas devdep

### PR DESCRIPTION
**Pull request by rjain15**
The canvas dev dependencies were creating a dependency on cairo, which resulted in a system library to be installed on macOSX. Here is the issue created for this dependency. I have removed the canvas dependency and checked the example programs and they seem to work. I am creating this pull request, but I am not sure what was the reason for this dependency. 

https://github.com/jerairrest/react-chartjs-2/issues/207

### CODE REVIEW

**Comment by jerairrest** [comment]
canvas is needed for running the test suite. I feel like the canvas dependency can probably be removed but the prebuilt will need to stay to make CI pass. Please readd `canvas-prebuilt` and give that a try. 

Thanks again for you contribution! I really appreciate it.

File: package.json [hunk 1]
```diff
@@ -16,50 +16,48 @@
...
-    "babel-preset-stage-1": "^6.13.0",
-    "babel-register": "^6.18.0",
+    "babel-preset-es2015": "^6.24.1",
+    "babel-preset-react": "^6.24.1",
+    "babel-preset-stage-1": "^6.24.1",
+    "babel-register": "^6.26.0",
     "babelify": "^7.3.0",
     "brfs": "^1.4.3",
-    "canvas": "^1.6.2",
-    "canvas-prebuilt": "^1.6.5-prerelease.1",
```
Comment (jerairrest) [inline package.json]: Please add this back in

File: package.json [hunk 2]
```diff
@@ -16,50 +16,48 @@
...
-    "chart.js": "2.6.0",
-    "cross-env": "^5.0.0",
-    "css-loader": "^0.28.5",
-    "debug": "^2.4.1",
-    "enzyme": "^2.6.0",
-    "eslint": "^1.6.0",
-    "eslint-plugin-react": "^3.5.1",
-    "extract-text-webpack-plugin": "^3.0.0",
+    "chai": "^4.1.2",
+    "chart.js": "2.7.0",
```
Comment (benmccann) [inline package.json]: chart.js is on 2.7.1 now

Comment (skirunman) [reply package.json]: 2.7.2 now

**Comment by jerairrest** [comment]
This is two years old, closing

**Pull request closed without merging**